<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="oneButton">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>One</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoButton">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>48</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Two</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threeButon">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>84</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Three</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourButton">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>120</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Four</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fiveButton">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>156</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Five</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
